joda-time/src/main/java/org/joda/time/Period.java

```diff
@@ -767,7 +767,7 @@ public int getYears() {
      * @return the number of months in the period, zero if unsupported
      */
         public int getMonths() {
-    return getPeriodType().getIndexedField(this, PeriodType.MONTH_INDEX);    }
+        return getPeriodType().getIndexedField(this, PeriodType.MONTH_INDEX);    }
```

Comment:
The style here doesn't match the rest of the code. Please revise the PR (do *not* close this PR and open a new one).